src/background.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Comment:
The background page is an invisible page, do we really need this `meta name="viewport"` here?

Reply:
no need actually. so removed :-)